src/common/pages/witnesses.tsx

```diff
@@ -144,6 +152,31 @@ class WitnessesPage extends BaseComponent<PageProps, State> {
         this.stateSet({witnessVotes: newVotes});
     }
 
+    getWitness = async (witnessArray:WitnessTransformed[])=>{
+        const witnessUserNamesArray:string[] =
+            witnessArray.map((item: WitnessTransformed)=>{
+            return item.name
+        })
+        try {
+            const accounts:FullAccount[] = await getAccounts(witnessUserNamesArray);
+            const byWitnessState:WitnessTransformed[] = witnessArray.map((item: WitnessTransformed, index: number) => {
+                try {
+                    const parsedArray = JSON.parse(accounts[index].json_metadata ?accounts[index].json_metadata:'');
+                    return {
+                        ...item,
+                        witnessBy:parsedArray.profile.name ? parsedArray.profile.name: undefined
```

Comment:
Not `name` should be parsed, but `witness_owner` field should be parsed and not all witnesses have that field.